dulwich/errors.py

```diff
@@ -164,6 +164,10 @@ class ObjectFormatException(FileFormatException):
     """Indicates an error parsing an object."""
 
 
+class CorruptedFormatException(FileFormatException):
+    """Indicates a corruption in the object's disk representation."""
```

Comment:
Any FileFormatException is a sign of an invalid disk representation; perhaps you mean something like EmpyFileException?